Task 19.2.py

```diff
@@ -0,0 +1,8 @@
+def in_range(start, end, step=1):

```

Comment:
Please update the function to match the behaviour of the built-in `range` (positional argument only, it can also use one, two or three arguments)